<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>471</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="Box_Point_ID">
      <property name="text">
       <string>##</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="Box_Point_CS">
      <property name="text">
       <string>##</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Coordinate System:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Box_Point_X">
          <property name="text">
           <string>##</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Box_Point_Y">
          <property name="text">
           <string>##</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Box_Point_Z">
          <property name="text">
           <string>##</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="Button_SetPos">
        <property name="text">
         <string>Set Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_Del">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
